grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -286,13 +287,14 @@ class LoadDataService {
         // 2b. If there are no available items:
         //   - then no snapshot is being saved
         if (availableItems.size() && isInventoryBaselineEnabled) {
-            inventoryImportProductInventoryTransactionService.createInventoryBaselineTransactionForGivenStock(
-                    targetWarehouse,
-                    null,
-                    products,
-                    availableItems,
-                    inventoryBaselineTransactionDate
+            InventoryBaselineTransactionCommand baselineTransactionCommand = new InventoryBaselineTransactionCommand(
+                    facility: targetWarehouse,
+                    sourceObject: null,
+                    products: products,
+                    availableItems: availableItems,
+                    transactionDate: inventoryBaselineTransactionDate
             )
+            inventoryImportProductInventoryTransactionService.createInventoryBaselineTransactionForGivenStock(baselineTransactionCommand)
```

Comment:
@ewaterman you were the one that implemented the `createInventoryBaselineTransactionForGivenStock` method - FYI - I needed to add another param (`transactionSource`) and since this method has already had a lot of arguments, I decided to make a command object to have only one param.
Please, double check me if I refactored it correctly in all of the places.